integrationExamples/gpt/userId_example.html

```diff
@@ -109,6 +109,15 @@
             googletag.pubads().disableInitialLoad();
         });
 
+        const eidsProvider = function(){
+            return [{
+                source: "prov.com",
```

Comment:
where did prov.com come from?  I'm not saying its wrong just want to understand where it came from.

Reply:
oh I see its short hand for "provider" lets spell that out as "provider.com"